easystacks/software.eessi.io/2023.06/icelake_cclake/072-eb-5.0.0.yml

```
@@ -0,0 +1,23 @@
+# 072-eb-5.0.0.yml: total build duration = ?? minutes
+easyconfigs:
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/MUSCLE/5.1.0-GCCcore-12.3.0/easybuild/MUSCLE-5.1.0-GCCcore-12.3.0.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/MUSCLE/5.1.0-GCCcore-12.3.0/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Gblocks/0.91b/easybuild/Gblocks-0.91b.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Gblocks/0.91b/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/RAxML/8.2.13-gompi-2023a-avx2/easybuild/RAxML-8.2.13-gompi-2023a-avx2.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/RAxML/8.2.13-gompi-2023a-avx2/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/bcgTree/1.2.1-foss-2023a/easybuild/bcgTree-1.2.1-foss-2023a.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/bcgTree/1.2.1-foss-2023a/easybuild/reprod/easyblocks/*.py
+  - Java-21.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Java/21.0.5/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Java/21.0.5/easybuild/Java-21.0.5.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Java/21.0.5/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Nextflow/24.10.2/easybuild/Nextflow-24.10.2.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Nextflow/24.10.2/easybuild/reprod/easyblocks/*.py
```

I think you just need the first, the others have already been merged
```suggestion
  - Java-21.eb:
      options:
        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Java/21.0.5/easybuild/reprod/easyblocks/*.py
```

---

Ah, I thought you had issues with that _because_ the JavaRC wasn't there? But I guess not, because indeed I see them :) I'll remove them from here (even if a build of this PR would just skip them, so no huge deal)

---

The RC file is there, but the contents are not the same. For icelake and cascadelake `Java/21` is missing